source/server/config_validation/filesystem.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include "envoy/filesystem/filesystem.h"
+#include "envoy/server/options.h"
+
+namespace Envoy {
+namespace Filesystem {
+
+class WrapInstance : public Instance {
```

Comment:
Please add a short comment that describes this class's purpose (and Null/RestrictedInstance).

Reply:
+1